paddle/operators/math/im2col.cc

```diff
@@ -0,0 +1,244 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include "paddle/operators/math/im2col.h"
+
+namespace paddle {
```

Comment:
use name space as follows? 
```
namespace paddle {
namespace functor {

}// namespace functor
}// namespace paddle
```

The namespace in math is 

```
namespace paddle {
namespace operators {
namespace math {

}  // namespace math
}  // namespace operators
}  // namespace paddle
```

Reply:
Use the second way(paddle->operators->math).